book/IMaEA.md

```diff
@@ -6035,7 +6035,7 @@ Something has to change and here are some ideas about funding and recognition.
 
 1. GSoC give students a bounty.  That program could be duplicated by Adobe, Facebook, Microsoft, Apple and other Tech Giants.
 2. Large organizations could make donations to a foundation from which projects could apply for grants.
-3. Github could carry ads and share the revenue with open source projects.
+3. GitHub could carry ads and share the revenue with open source projects.
```

Comment:
I made a couple of brand fixes (this and JavaScript) and am adding them to my defaults, since they're both things I want to do generally and because they're good demos of what this tool can do. -- This is the second repository I've used them on (the first being its sibling).